projects/plugins/crm/tests/php/automation/events/class-event-trigger-test.php

```diff
@@ -0,0 +1,92 @@
+<?php // phpcs:ignore WordPress.Files.FileName.InvalidClassFileName
+
+namespace Automattic\Jetpack\CRM\Automation\Tests;
+
+use Automattic\Jetpack\CRM\Automation\Automation_Engine;
+use Automattic\Jetpack\CRM\Automation\Automation_Workflow;
+use Automattic\Jetpack\CRM\Automation\Triggers\Event_Deleted;
+use Automattic\Jetpack\CRM\Automation\Triggers\Event_New;
+use WorDBless\BaseTestCase;
+
+require_once __DIR__ . '../../tools/class-automation-faker.php';
+
+/**
+ * Test Automation's event triggers
+ *
+ * @covers Automattic\Jetpack\CRM\Automation
```

Comment:
Although this isn't 100% incorrect, then the way I understand the convers annotation is that this basically says "Tests everything in the Automation class" which doesn't exist 😛 

```suggestion
 * @covers Automattic\Jetpack\CRM\Automation\Triggers\Event_Deleted
 * @covers Automattic\Jetpack\CRM\Automation\Triggers\Event_New
```